python/fledge/common/iprpc.py

```diff
@@ -18,28 +18,20 @@
 import io
 import subprocess
 import threading
-
-import time
-
 import traceback
-
-import json
 import pickle
-import re
 import tempfile
-
 import time
 import logging
-
 import fcntl
-from utils import eprint
+
+from fledge.common.utils import eprint
```

Comment:
let's make this a relative path. the more complex the path is, the less flexible the module is.